{{cookiecutter.project_name}}/Makefile

```diff
@@ -24,15 +24,33 @@ for line in sys.stdin:
 endef
 export PRINT_HELP_PYSCRIPT
 
-BROWSER := python -c "$$BROWSER_PYSCRIPT"
+# Mandatory tools
+PIP := $(shell command -v pip 2> /dev/null)
+PYTHON := $(shell command -v python 2> /dev/null)
```

Comment:
This will give no guarantee that you will have `python3` available. Since you would like to make a virtualenv based on python3 it is worth to change these two command checks to `pip3` and `python3` explicitly.